testing/xbps/APKBUILD

```diff
@@ -0,0 +1,61 @@
+# Contributor: André Klitzing <aklitzing@gmail.com>
+# Maintainer: André Klitzing <aklitzing@gmail.com>
+
+pkgname=xbps
+pkgver=0.51
+pkgrel=0
+pkgdesc="XBPS to bootstrap VoidLinux"
```

Comment:
the package description should be more accurate, while it is true that it is mainly meant for bootstrapping voidlinux, that doesn't necessarily have anything to do with the package itself